Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10-alpine
+WORKDIR /usr/app/src
+COPY . .
+EXPOSE 3000
+RUN apk add --no-cache --virtual .gyp python make g++
```

Comment:
Do we need to this now that the dependency has been updated/removed?

Reply:
No I guess we don't need it anymore